preprocessing/training/01_quality_improvment_processing_pipeline/slurm/cleaning_filtering/00_test_filter_user_titles.sh

```diff
@@ -1,7 +1,7 @@
 conda activate datacatalog
 
-TRAINING_DATA_PREPARATION_REPO=$WORK/code/big_science/data-preparation/preprocessing/training
-pushd $TRAINING_DATA_PREPARATION_REPO
+WORKING_DIR=$WORK/code/big_science/data-preparation/preprocessing/training/01_quality_improvment_processing_pipeline
```

Comment:
`improvement` instead of `improvment`

Reply:
Good catch! Done in 17c784adb9c4bb14be7a60e2d8c97b8b2e2813bf